pages/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, Heading, Text } from '@chakra-ui/react';
+import Body from '../layouts/Body';
+
+function Index() {
+  return (
+    <Body title="Trending">
+      <Box title="Home">
```

Comment:
Is this title really different from the one above?

Reply:
the correct one is the `props ` for the body component

```
<Body title="Trending">
.....
</Body>
```

to make this 

![image](https://user-images.githubusercontent.com/31835484/158060056-a06520ea-dccc-476a-9a05-ab5463fd6cfe.png)

i will delete for title props in BOX